.github/actions/get-jtreg/action.yml

```diff
@@ -39,36 +39,12 @@ runs:
       with:
         var: JTREG_VERSION
 
-    - name: 'Check cache for JTReg'
-      id: get-cached-jtreg
-      uses: actions/cache@v4
+    - name: 'Download JTReg artifact'
+      id: download-jtreg
+      uses: actions/download-artifact@v4
       with:
+        name: bundles-jtreg-${{ steps.version.outputs.value }}
         path: jtreg/installed
-        key: jtreg-${{ steps.version.outputs.value }}
-
-    - name: 'Checkout the JTReg source'
-      uses: actions/checkout@v4
-      with:
-        repository: openjdk/jtreg
-        ref: jtreg-${{ steps.version.outputs.value }}
-        path: jtreg/src
-      if: steps.get-cached-jtreg.outputs.cache-hit != 'true'
-
-    - name: 'Build JTReg'
-      run: |
-        # If runner architecture is x64 set JAVA_HOME_17_X64 otherwise set to JAVA_HOME_17_arm64
```

Comment:
Intentional simplification: since we no longer build JTReg on actual platforms, we can rely on current `ubuntu-22.04` runner to be Linux x86_64. This also dodges the transient headache with building JTreg on different (broken) MSYS envs on Windows, if you remember that mess.

Reply:
Right. *doh* I don't know what kind of brain meltdown I had there...